ec/ecresource/deploymentresource/schema_enteprise_search.go

```diff
@@ -66,18 +70,20 @@ func newEnterpriseSearchResource() *schema.Resource {
 func enterpriseSearchTopologySchema() *schema.Schema {
 	return &schema.Schema{
 		Type:     schema.TypeList,
-		Required: true,
+		Optional: true,
+		Computed: true,
```

Comment:
Looks like these should have the `Description` fields set as well.

Reply:
Probably not something to fix in this PR